### ISSUE TITLE
Godot 4.3 + iOS: Trying to load universal fmod binary

### ISSUE THREAD

**Issue by deevus** [utopia-rise/fmod-gdextension#247]
I am trying to add fmod to an existing Android/iOS project and getting the following error message when trying to run on iOS:

```
ERROR: GDExtension dynamic library not found: addons/fmod/libs/ios/libGodotFmod.ios.template_release.universal.dylib
at: open_library (core/extension/gdextension.cpp:801)
```

This file does not exist. Why is it looking for that and not the `arm64` lib?

Device: Apple MacBook M1 Pro
Architecture: Apple Silicon
Godot version: 4.3

**Comment by deevus** [comment]
Steps to reproduce:

- Create new project using Godot 4.3
- Add and enable fmod plugin
- Export to iOS project
- Run on device

**Comment by deevus** [comment]
Ok this looks like a simple issue with the gdextension file. It is referencing `addons/fmod/libs/ios/libGodotFmod.ios.template_release.universal.dylib` but what is actually shipped is `addons/fmod/libs/ios/libGodotFmod.ios.template_release.arm64.dylib`

**Comment by deevus** [comment]
However, I get this when trying to ship to the App Store:

<img width="746" alt="image" src="https://github.com/user-attachments/assets/31f00769-5623-4c9a-b7ba-f9e9791e6ae7">

**Comment by piiertho** [comment]
Hello ! It seems that's two bug on our own.
Here we expect universal binary.

https://github.com/utopia-rise/fmod-gdextension/blob/master/demo/addons/fmod/fmod.gdextension

I think we should add universal binary creation in CI.

The second bug is that our info.plist seems to not have the right iOS version in it.

I'll be pretty busy this week, so will not be able to fix in the next days.
For now you can modify both fmod.gdextension, to use arm64 binary, and info.plist to specify right iOS version.

**Comment by piiertho** [comment]
Oops sorry, I was confusing with macOS for info.plist
Do you have same problem on 4.2.2 ?
Maybe we just need to create a release with 4.3 godot-cpp.
Maybe they changed min iOS version in sconstruct to match info.plist

**Comment by deevus** [comment]
I see. I haven't had a chance to revisit this. Is there likely to be a build targeting 4.3 soon?

**Comment by piiertho** [comment]
Sorry, forgot to mention, this has been fixed in #274 
So on next release the problem won't be here anymore.

**Comment by PhoenixStroh** [comment]
That's great to hear! When do you think the next release will be?
